hw/ip/otbn/lint/otbn.vlt

```diff
@@ -16,3 +16,14 @@ lint_off -rule WIDTH -file "*/rtl/otbn_lsu.sv" -match "*'addr' generates 3 bits*
 // generates 4 bits.
 lint_off -rule WIDTH -file "*/rtl/otbn_stack.sv" -match "*'stack_wr_ptr_q' generates 4 bits*"
 
+// Operator EQ expects 32 bits on the LHS, but LHS's VARREF 'rd_addr_a_i'
+// generates 5 bits.
+lint_off -rule WIDTH -file "*/otbn_rf_base.sv" -match "*'rd_addr_a_i' generates 5 bits*"
+
+// Operator EQ expects 32 bits on the LHS, but LHS's VARREF 'rd_addr_b_i'
+// generates 5 bits.
+lint_off -rule WIDTH -file "*/otbn_rf_base.sv" -match "*'rd_addr_b_i' generates 5 bits*"
+
+// Operator EQ expects 32 bits on the LHS, but LHS's VARREF 'wr_addr_i'
+// generates 5 bits.
+lint_off -rule WIDTH -file "*/otbn_rf_base.sv" -match "*'wr_addr_i' generates 5 bits*"
```

Comment:
Let's please address those in RTL, it's easier than having waivers for all possible tools (AscentLint will come next).

Reply:
These all come from my slight distaste of having to explicitly bit select on parameters (it's a constant, that the tool knows, why can't it only complain if said constant is outside of the range allowed by the width, grumble, grumble).

But yes these lint waivers aren't great, I was going to look at different ways of dealing with it in another PR that would also remove the catch all rule we've got (ignore unused signals) that still remains from early development.

I'll add explicit bit selects to the parameters to deal with it and loose the waivers.